test/Bootstrapper.FunctionalTests/BootstrapperTests.cs

```diff
@@ -151,19 +157,25 @@ public void BootstrapperInvokesAssemblyWithInferredAppBaseAndLibPathOnClr(string
             var outputFolder = flavor == "coreclr" ? "dnxcore50" : "dnx451";
 
             using (var runtimeHomeDir = TestUtils.GetRuntimeHomeDir(flavor, os, architecture))
+            using (var tempSamplesDir = BootstrapperTestUtils.PrepareTemporarySamplesFolder(runtimeHomeDir))
             using (var tempDir = TestUtils.CreateTempDir())
             {
-                var samplesPath = TestUtils.GetSamplesFolder();
-                var sampleAppRoot = Path.Combine(samplesPath, "HelloWorld");
+                var sampleAppRoot = Path.Combine(tempSamplesDir, "HelloWorld");
 
+                string stdOut, stdErr;
                 var exitCode = DnuTestUtils.ExecDnu(
                     runtimeHomeDir,
                     subcommand: "build",
-                    arguments: string.Format("{0} --configuration=Release --out {1}", sampleAppRoot, tempDir.DirPath));
+                    arguments: string.Format("{0} --configuration=Release --out {1}", sampleAppRoot, tempDir),
+                    stdOut: out stdOut,
+                    stdErr: out stdErr);
 
-                Assert.Equal(0, exitCode);
+                if (exitCode != 0)
```

Comment:
Yes, the failure will be caught by later assertion. We are not testing `dnu restore` here so we don't assert on the exit code of `dnu restore`